packages/postgres-database/src/simcore_postgres_database/__init__.py

```diff
@@ -1,8 +1,4 @@
 from . import storage_models, webserver_models
 from .models.base import metadata
 
-__all__ = [
-    'metadata',
-    'webserver_models',
-    'storage_models'
-]
+__all__ = ["metadata", "webserver_models", "storage_models"]
```

Comment:
also oneline here...

Reply:
black measures horizontal and vertical spaces and decides whether it is inline or multiline